hathor/p2p/resources/mining_info.py

```diff
@@ -49,8 +49,9 @@ def render_GET(self, request):
             self._settings.P2PKH_VERSION_BYTE.hex() + 'acbfb94571417423c1ed66f706730c4aea516ac5762cccb8'
         )
         block = self.manager.generate_mining_block(address=burn_address)
+        block.init_static_metadata_from_storage(self._settings, self.manager.tx_storage)
```

Comment:
Yes, this could be improved in the future, but I don't think it's problematic for now. Please see https://github.com/HathorNetwork/hathor-core/pull/1014#discussion_r1715835009.

Reply:
It seems ok. Thanks!